data-prepper-plugins/otel-metrics-raw-processor/src/main/java/com/amazon/dataprepper/plugins/processor/otelmetrics/OtelMetricsRawProcessorConfig.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.otelmetrics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OtelMetricsRawProcessorConfig {
+
+    @JsonProperty("calculateHistogramBuckets")
```

Comment:
We are aiming to have Data Prepper use snake_case for all configurations to promote consistency. (It's not quite there yet). But, for new properties we want to use snake_case only. The plugin framework will automatically use snake_case. So removing these `@JsonProperty` annotations (or removing the value from them) will fix this.